src/autotype/CMakeLists.txt

```
@@ -1,5 +1,19 @@
-if(WITH_XC_AUTOTYPE)
-    if(UNIX AND NOT APPLE AND NOT HAIKU)
+# Main auto-type static library
+set(autotype_SOURCES
+        AutoType.cpp
+        AutoTypeAction.cpp
+        AutoTypeMatchModel.cpp
+        AutoTypeMatchView.cpp
+        AutoTypeSelectDialog.cpp
+        PickcharsDialog.cpp
+        WindowSelectComboBox.cpp)
+
+add_library(autotype STATIC ${autotype_SOURCES})
+target_link_libraries(autotype Qt5::Core Qt5::Widgets)
+
+# Platform specific auto-type implementations
+if(UNIX AND NOT APPLE AND NOT HAIKU)
+    if(WITH_X11)
         find_package(X11 REQUIRED COMPONENTS Xi XTest)
         find_package(Qt5X11Extras 5.2 REQUIRED)
```

Version spec presumably not needed since Qt 5.12 is minimum:

```suggestion
        find_package(Qt5X11Extras REQUIRED)
```